content/sensu-go/6.2/operations/deploy-sensu/secure-sensu.md

```diff
@@ -115,8 +115,8 @@ You can specify a certificate and key for the web UI separately from the API.
 To do this, add the `dashboard-cert-file` and `dashboard-key-file` parameters for backend SSL configuration in `/etc/sensu/backend.yml`:
 
 {{< code yml >}}
-dashboard-cert-file: "/path/to/ssl/cert.pem"
-dashboard-key-file: "/path/to/ssl/key.pem"
+dashboard-cert-file: "/etc/sensu/tls/cert.pem"
```

Comment:
this should be backend-1.pem

Reply:
I'm confused here -- you would only specify this parameter if you wanted to use a different certificate (i.e. *not* backend-1.pem), right?